rsconf/package_data/sirepo/nginx.conf.jinja

```diff
@@ -10,6 +10,21 @@ server {
     root {{ nginx.default_root }};
     ssl_certificate {{ nginx.tls_crt }};
     ssl_certificate_key {{ nginx.tls_key }};
+{% if 'jupyterhub_enabled' in sirepo %}
+    proxy_http_version 1.1;
```

Comment:
[these are in ngnix.conf](https://github.com/radiasoft/rsconf/blob/84db636618318686e142cb0ba13448896919d50f/rsconf/package_data/nginx/global.conf.jinja#L85) so you don't need them here.

Reply:
https://github.com/radiasoft/rsconf/pull/128/commits/0090bba595e55686c9e97eb30d38cf762f5612d2